src/shared/components/palette/complex/read-complex-field-collection-table.component.ts

```diff
@@ -112,6 +112,8 @@ export class ReadComplexFieldCollectionTableComponent extends AbstractFieldReadC
     return result;
   }
 
+  keepOriginalOrder = (a, b) => a.key;
```

Comment:
Is it needed? I have removed it and all tests are green.

Reply:
Yes this is needed to keep original order of the entries from the ordered returned by BE. Otherwise Angular and browser will reorder alphabetically.

Reply:
Tests added now.